unfoldingWord/translationCore
# Updated ULB

#### This pull request addresses:

- Fixed resource updater script.
- Downloaded ULB v11.
- Updated ULB v10 reference to v11 in codebase

#### How to test this pull request:

- Test with https://github.com/translationCoreApps/tC_Resources/pull/8
- Delete the `resources` folder from the `translationCore` folder in your user folder.
- Open tC and verify that the ULB bible is rendered.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2864)
